addons/l10n_it_edi_sdicoop/tests/test_edi_reverse_charge_xml.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+import logging
+from lxml import etree
+
+from odoo.tests import tagged
+from odoo.addons.l10n_it_edi_sdicoop.tests.test_edi_xml import TestItEdi
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestItEdiReverseCharge(TestItEdi):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        def get_tags(*tag_codes):
```

Comment:
Mmmh I don't really like that because it's not very verbose.

What do you think about changing tax_data and passing tuple instead of string for tags ?
```
...
'invoice_repartition_line_ids': rep_lines(
                (100, 'base', ('+03',)),
                (100, 'tax', ('+vj9',)),
                (-100, 'tax')),
...
```
Explicit is better than implicit no?